### PR TITLE
Fix typo in functions-and-recursion docs

### PR DESCRIPTION
**Pull request by peeley**
Change `This can be done by applied the` to `This can be done by applying the`

## 📚 Description

Probably a nitpick, but found this typo while first reading through the docs.

## 🔖 Changes

See above.

### CODE REVIEW

File: doc/content/documentation/functions-and-recursion.md [hunk 1]
```diff
@@ -99,11 +99,11 @@ Calls the function with the given arguments. The last argument must be a list of
...
+Sometimes it is required that a variable should pass to a function by reference. This can be done by applying the `:reference` metadata to the symbol.
 
 ```phel
 (fn [^:reference my-arr]
   (php/apush my-arr 10))
 ```
 
-Support for references is very limited in Phel. Currently, it only works for function arguments (except destructuring).
\ No newline at end of file
+Support for references is very limited in Phel. Currently, it only works for function arguments (except destructuring).
```
Comment (Chemaclass) [inline doc/content/documentation/functions-and-recursion.md:109]: What's the change in this line? 😆

**Review by Chemaclass — APPROVED** [review]
Thanks

File: doc/content/documentation/functions-and-recursion.md (hunk 1 above)
Comment (jenshaase) [reply doc/content/documentation/functions-and-recursion.md:109]: Newline is missing. But that no problem. I will merge it!

**Comment by jenshaase** [comment]
I merged it and updated the website. Thanks @peeley!

File: doc/content/documentation/functions-and-recursion.md (hunk 1 above)
Comment (Chemaclass) [reply doc/content/documentation/functions-and-recursion.md:109]: Yes, it make sense. I thought it might be that 🙈👌🏻

**Pull request merged**